wsmaster/codenvy-hosted-api-admin/src/main/java/com/codenvy/api/user/server/UserServicePermissionsFilter.java

```diff
@@ -61,15 +66,26 @@ protected void filter(GenericResourceMethod genericResourceMethod, Object[] argu
                 //public methods
                 return;
             case "create":
+                if (!licenseManager.canUserBeAdded()) {
```

Comment:
It'd better to move this check below to next one. Because it is not good to see `UNABLE_TO_ADD_ACCOUNT_BECAUSE_OF_LICENSE` message when self creation is not allowed.